test/test_bson_corpus.py

```diff
@@ -17,10 +17,17 @@
 import binascii
 import functools
 import glob
-import json
 import os
 import sys
 
+if sys.version_info[:2] == (2, 6):
+    try:
+        import simplejson as json
+    except ImportError:
+        import json
+else:
+    import json
+
```

Comment:
Done, though it looks like there are a few files that don't have this at the top. Do we even use this?

```
$ for file in `ls test/test_*.py`; do fgrep 'sys.path[0:0]' "${file}" >/dev/null || echo $file; done
test/test_collation.py
test/test_gridfs_bucket.py
test/test_raw_bson.py
test/test_read_concern.py
test/test_threads.py
```

We can also run them like `python setup.py test -s <tests>`.